spec/features/posts_spec.rb

```diff
@@ -0,0 +1,98 @@
+require 'rails_helper'
+
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** I think it would be better if you fix all your tests to pass even if they are not part of this milestone such as tests for your `controllers`, most of them are not passing.
![Screenshot from 2023-02-02 13-18-31](https://user-images.githubusercontent.com/83042742/216310931-90f9629b-6c51-439f-9112-13cf9903243a.png)

Reply:
> * [ **OPTIONAL** ] **IMPORTANT:** I think it would be better if you fix all your tests to pass even if they are not part of this milestone such as tests for your `controllers`, most of them are not passing.
>   ![Screenshot from 2023-02-02 13-18-31](https://user-images.githubusercontent.com/83042742/216310931-90f9629b-6c51-439f-9112-13cf9903243a.png)

Hi, @elisha2kyakpo1 
As you can see from my screenshots my tests are running and passing so I am really stuck here and time is not on my side. 
How do we resolve this?
![featues_specs](https://user-images.githubusercontent.com/5064641/216313621-a7853738-93f9-402c-80d3-b1476592a8c9.png)